AppCenter/AppCenter/Internals/Ingestion/MSIngestionProtocol.h

```diff
@@ -26,6 +26,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
```

Comment:
We can talk about this one offline. I, @clpolet, and @ElektrojungeMS discussed about it and decided not to rename in this PR. It will cause a huge refactoring in core so we might end up with keeping the name as it is.